src/redux/categories/categories.js

```diff
@@ -0,0 +1,32 @@
+const CHECK_STATUS = 'bookstore/books/CHECK_STATUS';
+const REMOVE_CATEGORY = 'bookstore/books/REMOVE_CATEGORY';
```

Comment:
- [ ] Kindly notice that there is no need for the remove category action you should remove it.